<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exoplanetsDialog</class>
 <widget class="QWidget" name="exoplanetsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exoplanets Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="BarFrame" name="TitleBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>530</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="stelWindowTitle">
        <property name="text">
         <string>Exoplanets Plug-in Configuration</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeStelWindow">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="updatesGroup">
         <property name="title">
          <string>Update exoplanets data from Internet</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="internetUpdatesCheckbox">
            <property name="text">
             <string>Update from Internet sources</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="lastUpdateLabel">
              <property name="text">
               <string>Last update:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QDateTimeEdit" name="lastUpdateDateTimeEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="updateFrequencyLabel">
              <property name="text">
               <string>Update frequency (hours):</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="updateFrequencySpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="nextUpdateLabel">
              <property name="text">
               <string>[next update info]</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>17</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="updateButton">
              <property name="text">
               <string>Update now</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="displayBox">
         <property name="title">
          <string>Settings for exoplanets</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="displayModeCheckBox">
            <property name="toolTip">
             <string>Plot all systems with exoplanets without labels</string>
            </property>
            <property name="text">
             <string>Enable display of distribution for exoplanets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="timelineModeCheckBox">
            <property name="toolTip">
             <string>Display exoplanets since their discovery</string>
            </property>
            <property name="text">
             <string>Enable timeline discovery of exoplanets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="habitableModeCheckBox">
            <property name="toolTip">
             <string>Display of the planetary systems, which contains the potential habitable exoplanets</string>
            </property>
            <property name="text">
             <string>Enable display of the potential habitable exoplanets only</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="displayShowDesignationsCheckBox">
            <property name="text">
             <string>Enable display of designations for exoplanets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="displayAtStartupCheckBox">
            <property name="text">
             <string>Enable display at startup</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="displayShowExoplanetsButton">
            <property name="text">
             <string>Show exoplanets button on toolbar</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Exoplanets marker color</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="exoplanetMarkerColor">
              <property name="toolTip">
               <string>Exoplanets marker color</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Marker color for potential habitable exoplanets</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="habitableExoplanetMarkerColor">
              <property name="toolTip">
               <string>Marker color for potential habitable exoplanets</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="temperatureScaleLabel">
              <property name="text">
               <string>Temperature scale:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="temperatureScaleComboBox">
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>120</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="restoreDefaultsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Restore defaults</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveSettingsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Save settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="diagramTab">
      <attribute name="title">
       <string comment="tab in plugin windows">Diagram</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="0" column="0" rowspan="2">
        <widget class="QGroupBox" name="diagramGroupBox">
         <property name="title">
          <string>Diagrams: scatter plot</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="labelX">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>X axis</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QCheckBox" name="checkBoxLogX">
                <property name="toolTip">
                 <string>Logarithmic scale</string>
                </property>
                <property name="text">
                 <string>log scale</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QComboBox" name="comboAxisX">
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="minX">
                <property name="inputMethodHints">
                 <set>Qt::ImhDigitsOnly</set>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelMinMaxX">
                <property name="text">
                 <string notr="true">-</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="maxX">
                <property name="inputMethodHints">
                 <set>Qt::ImhDigitsOnly</set>
                </property>
                <property name="placeholderText">
                 <string>max</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="labelY">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Y axis</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QCheckBox" name="checkBoxLogY">
                <property name="toolTip">
                 <string>Logarithmic scale</string>
                </property>
                <property name="text">
                 <string>log scale</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QComboBox" name="comboAxisY">
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="minY">
                <property name="inputMethodHints">
                 <set>Qt::ImhDigitsOnly</set>
                </property>
                <property name="placeholderText">
                 <string>min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelMinMaxY">
                <property name="text">
                 <string notr="true">-</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="maxY">
                <property name="inputMethodHints">
                 <set>Qt::ImhDigitsOnly</set>
                </property>
                <property name="placeholderText">
                 <string>max</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="3" column="0" rowspan="4" colspan="2">
           <widget class="QCustomPlot" name="customPlot" native="true"/>
          </item>
          <item row="8" column="0" colspan="2">
           <widget class="QPushButton" name="plotDiagram">
            <property name="text">
             <string>Plot</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="infoTab">
      <attribute name="title">
       <string comment="tab in plugin windows">Info</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutInfoBox">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="infoTextBrowser">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="openLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="websitesTab">
      <attribute name="title">
       <string comment="tab in plugin windows">Websites</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutWebsitesBox">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="websitesTextBrowser">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="openLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aboutTab">
      <attribute name="title">
       <string comment="tab in plugin windows">About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutAboutBox">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="aboutTextBrowser">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="openLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>external/qcustomplot/qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
